task_11/css/style.css

```diff
@@ -0,0 +1,719 @@
+}
+.content__wrapper .tone__wrapper .radio_custom label{
+    flex-direction: row;
+}
+@media screen and (max-width: 1200px),(min-resolution:270dpi){
+    .content__wrapper .tone__wrapper{
+        align-items: center;
+        padding: 0;
+    }
+}
+
+
+/*range block*/
+.content__wrapper .range__wrapper{
+    margin-top: 1em;
+    text-align: center;
+}
+.content__wrapper .range__wrapper ul{
+    display: flex;
+    justify-content: space-between;
+    text-transform: uppercase;
+    font-weight: 700;
+    margin-top: 0.5em;
+    color: #a5abc5;
+}
+.content__wrapper .range__wrapper fieldset{
+    margin-top: 2em;
+}
+.content__wrapper .range__wrapper legend{
+    text-align: left;
+    padding: 1em 3em;
+    margin-top: 2em;
+    border: 1px solid #e6e8ee;
+    position: relative;
+    max-width: 500px;
+}
+.content__wrapper .range__wrapper legend .legend-title{
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 90%;
+}
+.content__wrapper .range__wrapper legend .legend-text{
+    display: block;
+    font-size: 95%;
+    margin-top: 1em;
+    color: #aea5b8;
+}
+.content__wrapper .range__wrapper legend i{
+    position: absolute;
+    left: 1.5em;
+    top: 18px;
+}
+
+
+
+/*bottom form block*/
+.content__wrapper .content__bottom{
+    margin: 1em 0 0 0;
+}
+.content__wrapper .content__bottom p:not(.title){
+    padding: 1em 0;
+    color: #9fa5b8;
+}
+.content__wrapper .content__bottom p a{
+    padding: 1em 0;
+    color: #9fa5b8;
+}
+.content__wrapper .content__bottom .flex_row-start{
+    display: flex;
+    justify-content: flex-start;
+}
+.content__wrapper .content__bottom .flex_row-end{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-top: 4em;
+    flex-wrap: wrap;
+}
+.content__wrapper .content__bottom .flex_row-end label[for="agree"] a{
+    color: #6e768f;
+}
+.content__wrapper .content__bottom .flex_row-end .button__green{
+    margin: 0 1em;
+}
+@media screen and (max-width: 760px),(min-resolution:270dpi){
+    .content__wrapper .content__bottom .flex_row-end{
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+    }
+    .content__wrapper .content__bottom .flex_row-end .button__green{
+        margin: 1em 0;
+    }
+}
+.content__wrapper .content__bottom .flex_row-start .title{
+    display: block;
+    font-size: 85%;
+    margin-left: 0.5em;
+    text-transform: uppercase;
+    font-weight: 700;
+    color: #a5abc5;
+}
+.content__wrapper .content__bottom .flex_row-start div{
+    margin: 0 1em 0 0;
+}
+.content__wrapper .content__bottom .flex_row-start input:not([type="radio"]){
+    color: #6e768f;
+    border: 1px solid #d7d9e0;
+    border-radius: 3px;
+    padding: 0.5em 2.5em 0.5em 1em;
+}
+.content__wrapper .content__bottom .flex_row-start
+input:not([type="radio"])+label{
+    position: relative;
+    display: block;
+    width: 100%;
+    height: 100%;
+}
+.content__wrapper .content__bottom .flex_row-start
```

Comment:
Don't use such a long selector - it has a very big specificity level so it would be hard to overwrite them in future or inside @media query. Decrease it at least on 1 level.

Reply:
Near to end of task i lost in my own selectors and began to make a lot of nesting. Put order in selectors a little bit now.